site/static/examples/kind-with-registry.sh

```diff
@@ -43,13 +43,7 @@ for node in $(kind get nodes); do
 EOF
 done
 
-# 4. Connect the registry to the cluster network if not already connected
-# This allows kind to bootstrap the network but ensures they're on the same network
```

Comment:
(this comment explains why the script works this way)

we can expand this script to support podman but the current approach will break lots of people running this in clean CI environments

Reply:
with `--network bridge` as stated above, this section is not being removed